pset-functions.js

```diff
@@ -208,6 +334,12 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+function checkForDuplicates(str) {
+  let repeats = /(.)\1/;
```

Comment:
@anniecdavis27 nice regex! I had to look that one up :) ... the solution contains a non-regex alternative approach, you can check it out here:  https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L300